### DISCUSSION TITLE
Maui Blazor JS isolation error

### DISCUSSION THREAD

**Discussion by salghazaly** [dotnet/maui#6468] [Q&A]
I am working on a Blazor Maui app and I cannot load my js files for my components.
I have a separate project for my public components that are used by a blazor server app and a blazor MAUI app
for my component a js file for client side actions is isolated like following
 **protected override async Task OnAfterRenderAsync(bool firstRender)
        {
                if (firstRender)
                {
                        try
                        {
                                  Jsmodule = await JSRuntime.InvokeAsync<IJSObjectReference>(
                                  "import", "./_content/myAPP.Components/js/ComponentClient.js");
                        }
                        catch (Exception ex)
                        {
                                  Console.WriteLine(ex.Message);
                        }
                }
        }**

I put try catch but didn't get the exception the maui app doesn't work
although in the Blazor server app everything is working fine without any issues

**Comment by orosbogdan** [comment]
Trying the same with 
``` await JSHost.ImportAsync("Index","../Pages/Index.razor.js");```
on Window with MAUI Blazor App.
It says that the current platform does not support it.
